### PR TITLE
Multi-select long term + formatting

### PR DESCRIPTION
**Pull request by olliejth**
Added multiselect to long-term data, Changed some Streamlit formatting, added hidden monke(s)

### CODE REVIEW

File: dashboard/load_dashboard.py [hunk 1]
```diff
@@ -27,19 +29,32 @@
...
+
     water_count_bar = get_times_watered(longterm_df)
     max_temp_bar = get_max_temp_per_plant(longterm_df)
     min_moist_br = get_min_moisture_per_plant(longterm_df)
 
     st.altair_chart(water_count_bar, use_container_width=True)
     st.altair_chart(max_temp_bar, use_container_width=True)
     st.altair_chart(min_moist_br, use_container_width=True)
+
+    # st.title(":orangutan:" * 15)
```
Comment (Leon-JS02) [inline dashboard/load_dashboard.py:60]: Monke

File: dashboard/load_dashboard.py [hunk 2]
```diff
@@ -27,19 +29,32 @@
...
+        f":seedling: :green[Daily analytics ({date.today()})] :seedling:")
+
     st.altair_chart(botanist_plants_pie, use_container_width=True)
     st.altair_chart(temp_bar_chart, use_container_width=True)
     st.altair_chart(temp_line_chart, use_container_width=True)
     st.altair_chart(moist_bar_chart, use_container_width=True)
 
     # LONG TERM VISUALISATIONS:
 
+    st.header(":seedling: :green[Long-term analytics] :seedling:")
```
Comment (Leon-JS02) [inline dashboard/load_dashboard.py]: Nice changes to the style

File: dashboard/longterm_transform.py [hunk 3]
```diff
@@ -57,10 +55,9 @@ def get_min_moisture_per_plant(df: pd.DataFrame) -> alt.Chart:
 
     title = alt.TitleParams(
         'Min M experienced by each plant', anchor='middle')
```
Comment (Leon-JS02) [inline dashboard/longterm_transform.py:57]: Min M experienced? Should this be more descriptive?

File: dashboard/longterm_transform.py [hunk 4]
```diff
@@ -42,10 +41,9 @@ def get_max_temp_per_plant(df: pd.DataFrame) -> alt.Chart:
 
     title = alt.TitleParams(
         'Max T experienced by each plant', anchor='middle')
```
Comment (Leon-JS02) [inline dashboard/longterm_transform.py:43]: 'Max T experienced', I think this should say temperature.

**Review by Leon-JS02 — CHANGES_REQUESTED** [review]
This is solid code, would be nice with some changes to the graph titles

**Review by mehrdadhalali — APPROVED** [review]

**Review by Leon-JS02 — APPROVED** [review]

**Pull request merged**